app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CommentsController < ApplicationController
+  def new
+    @post = Post.find(params[:post_id])
```

Comment:
- [ ]  @cosywasswa Kindly consider removing this code, this is because you are only using it in the create method and this may lead to having a redundant code that doesn't have any use in our codebase.